src/components/layouts/Header.js

```diff
@@ -0,0 +1,34 @@
+// TODO : 재사용성으로 components로 분류했는데 공통 레이아웃 관련파일은 어떻게 분류?
+import { useNavigate } from "react-router-dom";
+import { ROUTE_PATH } from "../../constants/page";
+
+export default function Header() {
+  const navigate = useNavigate();
+
+  return (
+    <nav className="nav flex justify-around">
+      <div className="flex-center">
+        <h1
+          className="nav-title"
+          onClick={() => navigate(ROUTE_PATH.PRODUCT_LIST)}
+        >
+          CLEAN CODE SHOP
+        </h1>
+      </div>
+      <div className="flex gap-15">
+        <button
+          className="nav-button"
+          onClick={() => navigate(ROUTE_PATH.CART)}
+        >
+          장바구니
+        </button>
```

Comment:
`react-router-dom`의 `Link`를 사용하면 a 태그안에 children을 넣어줄 수 있어요.
onClick에 이벤트에서 navigate를 하는 방식보다 Link태그가 직관적으로 보이고 sementic하게 작성할 수 있는 부수효과도 있는 것 같아 추천드려요
```suggestion
        <Link className="flex-center" to={ROUTE_PATH.CART}>
          <button className="nav-button">장바구니</button>
        </Link>
```

Reply:
[7050032](https://github.com/next-step/react-shopping-cart/pull/29/commits/7050032b0f7d26450006d37250ce69bb02200d8c)